### DISCUSSION TITLE
oobabooga_windows\text-generation-webui\modules\ui_model_menu.py”, line 195, in load_model_wrapper

### DISCUSSION THREAD

**Discussion by timastras** [oobabooga/text-generation-webui#3778] [Q&A]
I am getting these 3 errors when I am trying to download or load a local model. On windows machine, any insight would be helpful

![image](https://github.com/oobabooga/text-generation-webui/assets/104849576/89e6b3c7-8b46-4be7-9802-53e71a4bd446)

**Comment by fahadh4ilyas** [comment]
Downloading that model need access to your huggingface username and huggingface password (or your huggingface access token). If you start it using `python server.py`, add `HF_TOKEN=<your_access_token>` before it to set environment variable for huggingface access token

**Comment by timastras** [comment]
I have already set my environment variables
